opflexagent/test/test_endpoint_file_manager.py

```diff
@@ -0,0 +1,569 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
This seems to have a lot of the same test code as test_gbp_ovs_agent.py. Perhaps you can add this file later when you are ready to refactor test_gbp_ovs_agent.py? Maintaining 2 copies will be a pain.